common/app/com/commercetools/sunrise/shoppingcart/CartFinderBySession.java

```diff
@@ -15,25 +14,25 @@
 import java.util.concurrent.CompletionStage;
 import java.util.function.UnaryOperator;
 
-import static com.commercetools.sunrise.shoppingcart.CartSessionUtils.overwriteCartSessionData;
 import static java.util.concurrent.CompletableFuture.completedFuture;
 
 public class CartFinderBySession implements CartFinder<Http.Session> {
```

Comment:
Yes, it is now. I think I didn't dare to replace the code that takes care of that with this because there were ongoing changes in that class at that moment. Now it's safe to be replaced.